Convert-ProwlarrSupportedIndexersToMarkdownTable.ps1

```diff
@@ -1,108 +1,194 @@
     .EXAMPLE
-    PS> .\Convert-ProwlarrSupportedIndexersToMarkdownTable.ps1 -Commit 1.1.1.1 -Build "test" -APIKey "asjdhfjashdf89787asdfsad87676"
+    PS> .\Convert-ProwlarrSupportedIndexersToMarkdownTable.ps1 -Commit 1.1.1.1 -AppAPIKey "asjdhfjashdf89787asdfsad87676"
 #>
 
 param (
-    [Parameter(
-        Mandatory, 
-        Position = 1
-    )]
+    [Parameter(Mandatory, Position = 1)]
     [string]
     $Commit,
-    [Parameter(
-        Mandatory, 
-        Position = 2
-    )]
+    [Parameter(Position = 2)]
     [string]
     $Build,
-    [Parameter(
-        Mandatory,
-        Position = 3
-    )]
+    [Parameter(Mandatory, Position = 3)]
     [string]
-    $APIKey,
-    [Parameter(
-        Position = 4
-    )]
+    $AppAPIKey,
+    [Parameter(Position = 4)]
     [System.IO.FileInfo]
     $OutputFile = ".$([System.IO.Path]::DirectorySeparatorChar)supported-indexers.md",
-    [Parameter(
-        Position = 5
-    )]
+    [Parameter(Position = 5)]
     [uri]
-    $BaseURL = "http://localhost:9696"
-)
-
-$url=($BaseURL.ToString().TrimEnd("/"))+'/api/v1/Indexer/schema'
-$headers=@{'X-Api-Key'=$APIKey}
-$R= Invoke-WebRequest -Uri $url -Headers $headers -ContentType 'application/json' -Method Get
-$exp_indexername= { IF ($_.IndexerUrls) {'['+$_.name+']('+($_.IndexerUrls[0])+')'+'{#'+$_.infoLink.Replace('https://wiki.servarr.com/prowlarr/supported-indexers#','')+'}'} Else {$_.name+'{#'+$_.infoLink.Replace('https://wiki.servarr.com/prowlarr/supported-indexers#','')+'}'}}
-$indexerobject= $R.Content | ConvertFrom-Json
-$tblobj= $indexerobject | Sort-Object -Property 'name'
-
-#$tbl_PubUse= $tblobj | Where-Object {($_.privacy -eq "public") -and ($_.protocol -eq "usenet")} | `
-#         Select-Object @{Name = 'Indexer'; Expression =$exp_indexername}, `
+    $AppBaseURL = 'http://localhost:9696'
+),
+
+## Gather Inputs & Variables
+### User Inputs
+#### Convert Params to match vars
+$app_baseUrl = $AppBaseURL
+$app_apikey = $AppAPIKey
+####
+
+## Start Variables
+### Application Details
+$app_api_version = 'v1'
+$app_api_path = '/api/'
+$app_api_endpoint_version = '/system/status'
+$app_api_endpoint_indexer = '/indexer/schema'
+$headers = @{'X-Api-Key' = $app_apikey }
+#### Github App Info
+$gh_app_org = 'Prowlarr'
+$gh_app_repo = 'Prowlarr'
+#$gh_app_branch = 'develop'
+## Disabled; API used for manual usage only
+### Wiki Details
+$wiki_link = 'https://wiki.servarr.com'
+$wiki_app_path = '/prowlarr'
+$wiki_page = 'supported-indexers'
+$wiki_bookmark = '#'
+### Page Formating
+$wiki_1newline = "`r`n"
+$wiki_2newline = "`r`n`r`n"
+#$wiki_encoding = 'utf8'
+### Github Details
+$gh_web = 'https://github.com'
+$gh_web_commit = 'commit/'
+#$gh_baseurl = 'https://api.github.com'
+#$gh_api_path = '/'
+#$gh_api_type = 'repos/'
+#$gh_api_branches = 'branches/'
+## Disabled; GH API used for manual usage only
```

Comment:
Instead of commenting it out, implement a switch in the parameters above and then have an if statement for any of the GitHub pieces that means that they aren't commented out and need uncommenting to make this bit work

Reply:
extremely unlikely it will ever be used in production; merely exists for development purposes;  either comment out or remove completely.